src/runtime/particle-execution-host.ts

```diff
@@ -121,7 +121,12 @@ export class ParticleExecutionHost {
     this.particles.push(particle);
     const apiPort = this.choosePortForParticle(particle);
     stores.forEach((store, name) => {
-      apiPort.DefineHandle(store, store.type.resolvedType(), name, store.storageKey.toString());
+      apiPort.DefineHandle(
+          store,
+          store.type.resolvedType(),
+          name,
+          store.storageKey.toString(),
+          particle.getConnectionByName(name).handle.ttl);
```

Comment:
Splitting these up into multiple lines is a travesty against readability :(

Reply:
undone